examples/events.js

```diff
@@ -66,5 +66,150 @@ export default [
     'title': 'Birthday Party',
     'start':new Date(2015, 3, 13, 7, 0, 0),
     'end': new Date(2015, 3, 13, 10, 30, 0)
-  }
+  },
+  ...today(1)
 ]
+
+function today(idx = 0) {
```

Comment:
by that I mean somewhere in: https://github.com/intljusticemission/react-big-calendar/tree/master/stories